dataland-frontend/src/utils/Formatter.ts

```diff
@@ -49,3 +49,20 @@ export function formatNumberToReadableFormat(value: number | undefined | null):
     maximumFractionDigits: 2,
   });
 }
+/**
```

Comment:
Almost duplicate to formatAmountWithCurrency

Reply:
Korrekt, aber so wie ich die Diskussion verstanden habe, soll in Zukunft das "Amount" in AmountWithCurrency umbenannt werden zu "Value" und sich damit auhc der Name ändern. Wir wollten jetzt hier nicht eine Component benutzen die wir in Zukunft dann wieder refactoren müssen und dafür ein Migrationskript schreiben müssen. 
DIe Alternative ist dass wir jetzt hier auch AmountWithCurrency benutzen und das in Zukunft wieder umbenennen.